### ISSUE TITLE
Evans asks to update when on latest version

### ISSUE THREAD

**Issue by danielyli** [ktr0731/evans#61]
Just saw this when starting Evans:

```
new update available:
  current version: 0.4.1
   latest version: 0.4.1
? update?
```

**Comment by ktr0731** [comment]
can you show me more describe the situation?
also, I want to know what the means you installed by.

**Comment by danielyli** [comment]
Yeah, of course.

On an Ubuntu machine (`Ubuntu 16.04.4 LTS (GNU/Linux 4.4.0-1057-aws x86_64)`), I install and run Evans by running:

```
$ curl -OL https://github.com/ktr0731/evans/releases/download/0.4.1/evans_linux_amd64.tar.gz
$ tar -xf evans_linux_amd64.tar.gz
$ ./evans --host localhost --port 50052 --package myservice --service MyService --path protos protos/myservice/MyService.proto

new update available:
  current version: 0.4.1
   latest version: 0.4.1

? update? (y/N)
```

**Comment by ktr0731** [comment]
thanks! I fixed at #64.
